bin/new-notification-config.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
```

Comment:
new-notification-config.js should be renamed to email-config.js

Reply:
Done